api/server/routes_create.go

```diff
@@ -89,9 +86,9 @@ func (s *Server) handleRouteCreate(c *gin.Context) {
 
 	}
 
-	route, err := s.Datastore.InsertRoute(ctx, wroute.Route)
+	route, err := s.Datastore.InsertRoute(c, wroute.Route)
 	if err != nil {
-		handleErrorResponse(c, err)
+		handleErrorResponse(c, r, err)
```

Comment:
Do internal methods need the RequestController?  Can't they just use the gin context?